web/src/beta/components/LayerSectionField/index.tsx

```diff
@@ -0,0 +1,67 @@
+import Icon from "@reearth/beta/components/Icon";
+import SidePanelSectionField from "@reearth/beta/components/SidePanelSectionField";
+import { SwitchField } from "@reearth/beta/hooks/useManageSwitchState/hooks";
+import { styled, useTheme } from "@reearth/services/theme";
+
+import LayerSectionItem, { LayerSectionItemProps } from "./item";
+
+export type LayerSectionFieldProps = {
+  layers: SwitchField<LayerSectionItemProps>[];
+  onClickLayerAdd?: () => void;
+  onActive?: (id: string) => void;
+  onAction?: (id: string) => void;
+  onVisible?: (id: string) => void;
+};
+
+const LayerSectionField: React.FC<LayerSectionFieldProps> = ({
+  layers,
+  onClickLayerAdd,
+  onActive,
+  onAction,
+  onVisible,
+}) => {
+  const theme = useTheme();
+
+  return (
+    <SidePanelSectionField title="Layers">
+      <LayerAddFrame>
+        <Icon
+          icon="layerAdd"
+          size={14}
+          color={theme.general.content.main}
+          onClick={onClickLayerAdd}
```

Comment:
Since the LayerAddFrame has padding, it would probably be nice to be able to click more than just the icon, so can you move the onClick to this component? 🙏🏻 
Can you also add a `:hover: { cursor: "pointer"; }` so we know its clickable